service/app/controllers/coursemanagement/validator/CourseBatchRequestValidator.java

```diff
@@ -45,10 +45,22 @@ public void validateUpdateCourseBatchRequest(Request request) {
             ERROR_CODE);
       }
     }
-    validateParam(JsonKey.NAME, ResponseCode.mandatoryParamsMissing, JsonKey.NAME);
+    if (request.getRequest().containsKey(JsonKey.NAME)
+        && StringUtils.isBlank((String) request.getRequest().get(JsonKey.NAME))) {
+      throw new ProjectCommonException(
```

Comment:
validateParam(String value, ResponseCode error, String errorMsgArgument) - this is sufficient for handling exception in this case. superclass has this method